docs/stats.md

```diff
@@ -0,0 +1,38 @@
+# Stats
+
+## Overview
+<!-- TODO: move module definitions here? -->
+A set of methods used to calculate statistics over the course of an
+experiment.
+Each such method is a higher-order function returning a `track_stat` method
+that takes in an Experiment object, an action key, and the address of the
+Keeper that executed said action.
```

Comment:
should also mention that each `track_stat` function is expected to record its statistic in the `state["stats"]` dict under a key equivalent to the higher-order function's name (e.g. `num_bids_placed` is the name of the HOF, and also we save the result to `state["stats"]["num_bids_placed"]`

or maybe this should go into Common Conventions below?